### ISSUE TITLE
Use vector images in your PDF

### ISSUE THREAD

**Issue by jolars** [TUMFTM/PointCloudCrafter#82]
I noticed that your images in the JOSS paper draft all use raster images, which I believe in most of these cases results in larger file sizes, text that's not readable by screen readers, and also just worse resolution and results when printing. Would you please consider rendering the images as pdfs instead before including them?

Ref: openjournals/joss-reviews#10459

**Comment by jolars** [comment]
Sorry, somehow managed to open this in the wrong repo!
